README.md

```diff
@@ -120,34 +163,63 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
 To run this code, use the following command: 
 <p align="center"> <code> cargo run --release </code> </p>
 
-Note that when running code that uses `tfhe-rs`, it is highly recommended
+> ℹ️ Note that when running code that uses `tfhe-rs`, it is highly recommended
```

Comment:
tfhe-rs -> TFHE-rs I guess to stick with the branding, seems it's an old mistake